packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -565,7 +567,9 @@ function storeS3BucketInfo(category: string, deploymentBucketName: string, envNa
   const amplifyMeta = stateManager.getMeta();
   const teamProviderInfo = stateManager.getTeamProviderInfo();
 
-  _.set(teamProviderInfo, [envName, 'categories', category, resourceName], { deploymentBucketName, s3Key });
+  const tpiResourceParams: $TSAny = _.get(teamProviderInfo, [envName, 'categories', category, resourceName], {});
+  _.assign(tpiResourceParams, { deploymentBucketName, s3Key });
```

Comment:
I'm not sure if this is the overwriting you were referring to in the PR description, but this is still going to overwrite these values in tpiResourceParams. Just making sure that's intentional?

Reply:
Intentional. Previously this code would overwrite additional parameters as well.